src/main/tsconfig.json

```diff
@@ -9,7 +9,8 @@
     "allowJs": false,
     "esModuleInterop": true,
     "strictNullChecks": true,
-    "strict": true
+    "strict": true,
+    "plugins": [{ "name": "typescript-plugin-css-modules" }]
```

Comment:
backend and shared don't need the css plugin, or does the LSP have a bug where it is needed in all modules/targets?